### ISSUE TITLE
✔️ 🧩 🐞 Bug Report: Elementor Pro breaks Imagify Media Library filters due to _elementor_is_screenshot meta filter (ED-8596)

### ISSUE THREAD

**Issue by ShadowXVII** [elementor/elementor#19988]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When looking at the Media LIbrary and filtering using Imagify, the `_elementor_is_screenshot` filter takes over and clobbers the meta query which breaks the other plugins meta query.

Perhaps the _elementor_is_screenshot should be additive instead of replacing meta query?

https://github.com/wp-media/imagify-plugin

### Query with Elementor Pro disabled

```
cache_results	1
comments_per_page	50
lazy_load_term_meta	1
meta_query	
Array
(
    [relation] => or
    [0] => Array
        (
            [key] => _imagify_status
            [value] => success
            [compare] => =
        )

    [1] => Array
        (
            [key] => _imagify_status
            [value] => already_optimized
            [compare] => =
        )

)
order	DESC
paged	1
post_mime_type	
Array
(
    [jpg|jpeg|jpe] => image/jpeg
    [png] => image/png
    [gif] => image/gif
    [pdf] => application/pdf
)
post_status	inherit,private
post_type	attachment
posts_per_page	99
update_post_meta_cache	1
update_post_term_cache	1
```

### Query with Elementor Pro enabled

```
cache_results	1
comments_per_page	50
lazy_load_term_meta	1
meta_query	
Array
(
    [0] => Array
        (
            [key] => _elementor_is_screenshot
            [compare] => NOT EXISTS
        )

)
order	DESC
paged	1
post_mime_type	
Array
(
    [jpg|jpeg|jpe] => image/jpeg
    [png] => image/png
    [gif] => image/gif
    [pdf] => application/pdf
)
post_status	inherit,private
post_type	attachment
posts_per_page	99
update_post_meta_cache	1
update_post_term_cache	1
```

### Steps to reproduce

1) Install imagify
2) Install elementor pro
3) Go to media library
4) Attempt to filter for Optimised or Error images
5) Note that query meta gets clobbered by _elementor_is_screenshot

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

`					
== Server Environment ==
	Operating System: Linux
	Software: nginx
	PHP Version: 8.1.10
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.2
	Permalink Structure: /blog/%year%/%monthnum%/%day%/%postname%/
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_AU
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36

== Active Plugins ==

	Elementor
		Version: 3.7.8
		Author: Elementor.com

	Elementor Blocks for Gutenberg
		Version: 1.1.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.7
		Author: Elementor.com

	Imagify
		Version: 2.0
		Author: Imagify - Optimize Images & Convert WebP

== Must-Use Plugins ==

== Elements Usage ==

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Inactive by default
	Import Export Template Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==
	

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	
`

**Comment by nicholaszein** [comment]
Hello @ShadowXVII! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hey @ShadowXVII, 

While reviewing your report, we noticed it was missing some key pieces of information.

👉 Please follow the guidelines for opening a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

⚠️ This report is missing key information: 

* System info

Without them, we simply can't help. 😕 

Please, share your System Info **here** so we can verify your report. **Do not send it via PM**, as we won't accept any PMs and may close your issue without any warning. If you're worried about sensitive information like site address, admin e-mail, and folder structure, simply remove it, but we need to know what is your website setup to be able to reproduce your issue.

Best regards,

Nick

**Comment by ShadowXVII** [comment]
@nicholaszein updated.

**Comment by Nevoss** [comment]
Hey, in the next pro release we will fix this issue, but there is a change that should be made by "Imagify" team, they set an 'or' relation in the `meta_query`, maybe they can nest this `or` relation like this:
```php
[
	[
		'relation' => 'or',
		[...SOME_QUERY],
		[...SOME_QUERY],
	]
	[...ELEMENTOR_QUERY]
]
```

for now even when we fix this issue on our side, we have to wait for them to fix in their side.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was solved with Elementor Pro **v3.10**! 🥳 

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
